app/models/Country.scala

```diff
@@ -66,4 +66,6 @@ object Country {
         )
     }
   }
+
+  val allCountries: Seq[Country] = euCountries // TODO: Awaiting confirmation of where to source this country list
```

Comment:
Assuming we need to wait for an answer on this?

Reply:
Yes, the question's been asked but no answer yet.